app-backend/project/Dependencies.scala

```diff
@@ -33,6 +33,7 @@ object Dependencies {
   val commonsIO          = "commons-io"                   % "commons-io"                        % Version.commonsIO
   val scopt              = "com.github.scopt"            %% "scopt"                             % Version.scopt
   val caffeine           = "com.github.ben-manes.caffeine" % "caffeine"                         % Version.caffeine
+  val scaffeine          = "com.github.blemale"          %% "scaffeine"                         % "2.0.0"
```

Comment:
We should probably follow the existing project convention here with dependency versions defined in `Version`.